src/main/java/trello/entities/Context.java

```diff
@@ -67,4 +80,31 @@ public Board getBoard() {
     public List getList() {
         return list;
     }
+
+    /**
+     * Sets context's user.
```

Comment:
Fix this comment please you set a List not user

Reply:
Solved